docs/templates/data_source_parent.rst.jinja

```diff
@@ -0,0 +1,101 @@
+===============================================================================
+{{ source.title }}
+===============================================================================
+
+.. list-table::
+   :widths: auto
+   :header-rows: 0
+   :stub-columns: 1
+
+   * - Source URL
+     - {{ source.path }}
+   {%- for field, description in source.description.items() %}
+   * - {{ field.replace("_", " ").title() }}
+     - {{ description }}
+   {%- endfor %}
+   * - Years Liberated
+     - {{ source.get_temporal_coverage() }}
+   * - Source Temporal Coverage
+     - {{ source.get_temporal_coverage(partitions=source.source_file_dict) }}
+   * - Size (Download)
+     - {{ source.source_file_dict.download_size_mb }} MB
+   * - PUDL Code
+     - ``{{ source.name }}``
+   * - Issues
+     - `Open {{ source.title }} issues <https://github.com/catalyst-cooperative/pudl/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3A{{ source.name }}>`__
+
+Background
```

Comment:
How do you feel about breaking up the narrative sections into a bunch of smaller blocks? Is there enough uniformity across the different data sources that it makes sense to have these be different fields which need to be filled out for each of them? Or do they not really apply clearly across everything? Would it make more sense to break the page up into two sections, one that's all the more programmatically generatable content, and another that's just a single large narrative block that needs to be written for each data source? 

Are we highlighting the right information about the data sources?  Are we presenting it in the right order (most valuable / salient first, pedantic details last?)

It might be good to have a blank template, with a description of what kind of information is supposed to go in each section that we're maintaining.